### ISSUE TITLE
Notation for Plainchant or Anglican Chant?

### ISSUE THREAD

**Issue by greyspectrum** [gbj/venite#215]
I found a TypeScript [library](https://github.com/0xfe/vexflow) for rendering musical notation. Would you be interested in including notation for plainchant, as well as Anglican chant, as options in Venite for those who prefer to chant the psalms?

I believe the [_Plainsong Psalter_](https://www.churchpublishing.org/products/plainsongpsalter) and the [_Anglican Chant Psalter_](https://www.churchpublishing.org/products/anglicanchantpsalter) have the appropriate pointing for the 1979 BCP Psalter. The [_St. Dunstan's Plainsong Psalter_](https://andrewespress.com/st-dunstans-plainsong-psalter-hardcover/) has pointing for the Coverdale Psalter. I'm not sure of the copyright status of the musical notation itself, however.

The Invitatory, the Cantlicles, the Apostles' Creed, the Lord's Prayer, and the Suffrages may also be set to plainchant.

I would be interested in working on this, if it is a feature you would like to include.

**Comment by gbj** [comment]
The primary issue with anything musical is always copyright. As far as I know even the basic notations included in _The Hymnal 1982_ are copyrighted and Church Pension is quite restrictive about access. I can't speak to the copyright holder for the Plainsong Psalter.

I'm very open to include this on the technical level, if you have a non-copyrighted version or permission to reproduce a copyrighted one for free.

**Comment by greyspectrum** [comment]
I found this: [Sing the Office](https://singtheoffice.com/). They wrote a short blog post outlining what they were trying to accomplish, [here](https://singtheoffice.substack.com/p/singtheoffice-second-edition).

They based it off of the [CTS Divine Worship Daily Office](https://www.ctsbooks.org/product/daily-office/), but pointed it for plainsong. The source text their using is very close to the 1662 Book of Common Prayer, put out by a Roman Catholic publisher for Catholics and others who are interested in praying the office in the Anglican style.

Since this is pretty much a perfect resource, it seems silly to try and reproduce their efforts from scratch, especially since, as far as I can tell, the Church Publishing copyright would be a hard stop impediment to introducing service music from the hymnal.

Of course, since this is basically the 1662 BCP, it's not approved for public worship in the Episcopal Church. It almost seems easier to get approval to use the 1662 BCP than it does to try and deal with the hymnal copyright.

For all of these reasons, it's probably best to close this issue, I would imagine. At the very least, I would find it hard to be motivated to put together a PR that essentially replicates something that is already out there and works.

**Comment by ordokr** [comment]
Did you end up including the music notation library?  I would be interested in implementing this repo on https://divineoffice.org/.  They have approval for all the music and of course the liturgy of the hours from the USCCB.  @gbj I know this is Anglican, but it would be a perfect fit for the Catholic office and readings!

**Comment by gbj** [comment]
No, there's no musical notation library included. You're welcome to use any of the code in this repo -- it's a bit of a legacy Angular app from a time when I was just getting back into programming and has largely been on the back burner for a few years, so caveat emptor.
